src/agent/debuglet.ts

```diff
@@ -597,6 +622,11 @@ export class Debuglet extends EventEmitter {
                 }
                 that.scheduleBreakpointFetch_(
                     that.config_.breakpointUpdateIntervalSec);
+                if (that.promise_) {
+                  that.promiseResolve_();
+                  that.promise_ = null;
```

Comment:
... and this should be `that.promiseResolve_ = null;`

Reply:
Done.